src/AppBundle/Entity/CommitteeFeedItem.php

```diff
@@ -98,11 +98,41 @@ public static function createEvent(
 
     public function getContent(): ?string
     {
+        if ($this->event instanceof CommitteeEvent) {
+            return $this->event->getDescription();
```

Comment:
What about adding `$this->content = $event->getDescription()` in `createEvent` above?